packages/account/src/Components/poi/status/unsupported/unsupported.jsx

```diff
@@ -28,7 +40,16 @@ const Unsupported = ({ country_code, handlePOIforMT5Complete, ...props }) => {
         country_code,
     });
 
-    if (props?.manual?.status === identity_status_codes.pending) return <UploadComplete />;
+    if (manual) {
+        if (manual.status === identity_status_codes.pending) return <UploadComplete />;
+        else if ([identity_status_codes.rejected, identity_status_codes.suspected].includes(manual.status)) {
+            if (!allow_poi_resubmission) return <Limited />;
```

Comment:
what happens when `manual_status` is rejected and `allow_poi_resubmission` is true? should'nt we allow client to resubmit?

Reply:
The default case handles that